.taskcluster.yml

```diff
@@ -229,7 +232,13 @@ tasks:
                               - bash
                               - -cx
                               - $let:
-                                    extraArgs: {$if: 'tasks_for == "cron"', then: '${cron.quoted_args}', else: ''}
+                                    extraArgs:
+                                        $if: 'tasks_for == "cron"'
+                                        then: '${cron.quoted_args}'
+                                        else:
+                                            $if: 'tasks_for == "github-release"'
+                                            then: '--head-tag="$MOBILE_HEAD_TAG"'
```

Comment:
We could pass this always (and it will end up passing an empty value when it isn't specified (which, IIRC, matches not passing it).